app/commands/exercise/representation/search.rb

```diff
@@ -0,0 +1,79 @@
+class Exercise::Representation::Search
+  include Mandate
+
+  # Use class method rather than constant for easier stubbing during testing
+  def self.requests_per_page = 20
+
+  initialize_with criteria: nil, status: nil, mentor: nil, track: nil, order: :most_submissions,
```

Comment:
Can this ever be correctly called without a `status`? If not, it shouldn't be `nil` and I feel like it should probably be a boolean.